src/applications/vre/28-8832/config/chapters/status-selection/wizard.js

```diff
@@ -0,0 +1,20 @@
+export const schema = {
+  type: 'object',
+  properties: {
+    status: {
+      type: 'string',
+      enum: [
+        'Active duty service member',
+        'Veteran',
+        'Spouse of a Veteran or service member',
+        'Child of a Veteran or service member',
+      ],
+    },
+  },
+};
+
+export const uiSchema = {
+  status: {
+    'ui:widget': 'radio',
```

Comment:
I think this needs a ui:title of `Select one of the options below.` It isn't reflected in the mockups right now, but it's possible this field is also required. Can you verify with Jim?